test/mpi/io/Makefile.am

```diff
@@ -30,6 +30,8 @@ noinst_PROGRAMS = \
     external32_derived_dtype \
     tst_fileview \
     zero_count \
+    large_count \
+    large_dtype \
```

Comment:
How big (in terms of file space and memory it needs) are these two tests?

Reply:
I don't know the memory footprint, but the output file size is 18 GB when running on 2 MPI processes.

Reply:
Just realized that both tests are not added to the testlist so they won't be run during CI testing. I guess that is fine for now.

Reply:
/usr/bin/time says `large_count` needs ~26 MiB (26276 KiB) of memory (max RSS)  -- that seems low.

Reply:
Makes sense. That is how the chunking algorithm is supposed to work, right?

Reply:
Wait, it allocates a local buffer of size ~4GB, so ~26MB does seem low